src-tauri/src/main.rs

```diff
@@ -62,22 +75,32 @@ fn main() {
     let _fake_xdim = "1000".to_string();
     let _fake_ydim = "1000".to_string();
 
+    // Hardcoded VA and HA
+    let _fake_va = "186".to_string();
+    let _fake_ha = "186".to_string();
+
+
     // UNCOMMENT TO CALL PIPELINE WITH HARDCODED DATA
-    // let _result = pipeline(
-    //     _FAKE_RADIANCE_PATH.to_string(),
-    //     _FAKE_HDRGEN_PATH.to_string(),
-    //     _fake_output_path,
-    //     _fake_temp_path,
-    //     _fake_input_images,
-    //     _fake_response_function,
-    //     _fake_fisheye_correction_cal,
-    //     _fake_vignetting_correction_cal,
-    //     _fake_diameter,
-    //     _fake_xleft,
-    //     _fake_ydown,
-    //     _fake_xdim,
-    //     _fake_ydim,
-    // );
+    let _result = pipeline(
```

Comment:
I think we should probably comment out the hardcoded backend-only test before merging with main since we've got the frontend set up now